server/main.js

```diff
@@ -1,8 +1,11 @@
 const app = require('./server')
 const PORT = process.env.PORT || 3000
+const { db } = require('./db/index')
 
 //const require db
 
+db.sync()
+
```

Comment:
db.sync() should be in a try catch inside of app.listen():

```suggestion
app.listen(PORT, async () => {
    try {
        await db.sync()
        console.log(`Listening on ${PORT}`)
    } catch (error) {
        console.log(error)
    }
})

```

Reply:
updated